src/pages/_app.tsx

```diff
@@ -100,11 +100,7 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
       <Head>
         <meta name="google-site-verification" content="CDEVFlJTyVZ2vM7ePugKgWsl_7Rd-MrfDv42u0vZ0B0" />
         <link rel="icon" href="favicon.ico" />
-        <link
-          rel="canonical"
-          href={`${process.env.NEXT_PUBLIC_HOSTNAME}/near/widget/NearOrg.HomePage`}
-          key="canonical"
-        />
+        <link rel="canonical" href={`${process.env.NEXT_PUBLIC_HOSTNAME}${router.asPath}`} key="canonical" />
```

Comment:
@s-n-park do we need a `/` in between these two variables? That pattern is used in the diff above

Reply:
actually, the preview looks ok.